plugin-server/src/cdp/metrics/metrics.ts

```diff
@@ -0,0 +1,37 @@
+import { Counter, Gauge, Histogram } from 'prom-client'
+
+export const histogramKafkaBatchSize = new Histogram({
```

Comment:
I actualy don't love this tbh... We had this before and it made it really hard to know where that metric was used. I think these would be better co-located with the thing using it